codalab/lib/server_util.py

```diff
@@ -7,10 +7,39 @@
 import sys
 import urllib
 
-from bottle import request, HTTPResponse, redirect
+from bottle import abort, request, HTTPResponse, redirect, app, local
 from oauthlib.common import to_unicode, bytes_type
 
 
+def query_get_bool(key, default=False):
+    value = request.query.get(key, None)
```

Comment:
It's OK, rebasing is tough, but really helps with getting reviews done quickly, at least in my experience.